doc/installing-from-release.md

```diff
@@ -18,29 +18,48 @@ Make sure you've got one of Rayhunter's [supported devices](./supported-devices.
     cd ~/Downloads/rayhunter-vX.X.X-PLATFORM
     ```
 
-4. Turn on your device by holding the power button on the front.
+   On Windows you can decompress using the file browser, then navigate to the
+   folder that contains `installer.exe`, **hold Shift**, Right-Click inside the
+   folder, then click "Open in PowerShell".
 
-   * For the Orbic, connect the device using a USB-C cable.
-     * Or connect to the network if using the network based installer, this is especially recommended on Windows.
-   * For TP-Link, connect to its network using either WiFi or USB Tethering.
+4. **Connect to your device.**
 
-5. Run the installer:
+   First turn on your device by holding the power button on the front.
 
-    ```bash
-    # On MacOS, you must first remove the quarantine bit
-    xattr -d com.apple.quarantine installer
-    ```
-    Then run the installer:
-    ```bash
-    ./installer orbic
-    # or: ./installer [orbic-network|tplink|tmobile|uz801|pinephone|wingtech]
-    ```
+   Then connect to the device using either WiFi or USB tethering.
+
+   You know you are in the right network when you can access
+   <http://192.168.1.1> (Orbic) or <http://192.168.0.1> (TP-Link) and see the
+   hardware's own admin menu.
+
+5. **On MacOS only**, you have to run `xattr -d
+   com.apple.quarantine installer` to allow execution of
+   the binary.
+
+6. **Run the installer.**
+
+   ```bash
+   # For Orbic:
+   ./installer orbic --admin-password 'mypassword'
```

Comment:
We should mention the USB installer here for people who still want to install that way.